omnisafe/adapter/early_terminated_adapter.py

```diff
@@ -12,37 +12,48 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
-"""OnPolicy Adapter for OmniSafe."""
+"""EarlyTerminated Adapter for OmniSafe."""
 
 from __future__ import annotations
 
+from typing import Any
+
 import torch
 
 from omnisafe.adapter.onpolicy_adapter import OnPolicyAdapter
 from omnisafe.utils.config import Config
 
 
 class EarlyTerminatedAdapter(OnPolicyAdapter):
-    """OnPolicy Adapter for OmniSafe."""
+    """EarlyTerminated Adapter for OmniSafe."""
 
     def __init__(self, env_id: str, num_envs: int, seed: int, cfgs: Config) -> None:
         assert num_envs == 1, 'EarlyTerminatedAdapter only supports num_envs=1.'
 
         super().__init__(env_id, num_envs, seed, cfgs)
 
+        self._cost_limit: float
+        self._cost_logger: torch.Tensor
         self._cost_limit = cfgs.algo_cfgs.cost_limit
         self._cost_logger = torch.zeros(self._env.num_envs)
```

Comment:
You do not need to split the type annotation and assignment into two lines.

```suggestion
        self._cost_limit: float = cfgs.algo_cfgs.cost_limit
        self._cost_logger: torch.Tensor = torch.zeros(self._env.num_envs)
```

If you define a attribute outside of `__init__`, you should annotate it with:

```python
class MyClass:
    _logger: Logger

    def __init__(self, ...):
        ...

        self._init_logger()

    def _init_logger(self):
        self._logger = Logger(self.config)
```

Reply:
I see, I just misunderstand ``# annotate without an assignment``, I will fix all mistakes like that in follow commits.